url/urltestdata.json

```diff
@@ -5644,5 +5644,15 @@
     "input": "non-special://[:80/",
     "base": "about:blank",
     "failure": true
+  },
+  {
+    "input": "blob:https://example.com:443/",
```

Comment:
These tests are missing all the other fields, which causes them to fail in the WPT test runner. You need base, href, username, password, host, hostname, port, search, and hash.